### PR TITLE
[JetNews] Remove Hardcoded Strings

### PR DESCRIPTION
**Pull request by pandubaraja**
### JetNews

1. Removing hardcoded strings
2. Refactor Sections Enum accepting StringResId

Issue: https://github.com/android/compose-samples/issues/612

### CODE REVIEW

File: JetNews/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -28,4 +28,25 @@
     <string name="fewer_stories">Show fewer stories like this?</string>
     <string name="fewer_stories_content">This feature is not yet implemented</string>
     <string name="agree">AGREE</string>
+    <string name="close">CLOSE</string>
+
+    //Home Screen
```
Comment (chrisbanes) [inline JetNews/app/src/main/res/values/strings.xml:33]: This isn't a valid XML comment

File: JetNews/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -28,4 +28,25 @@
...
+    <string name="home_title">Home</string>
+    <string name="home_tap_to_load_content">Tap to load content</string>
+    <string name="home_top_section_title">Top stories for you</string>
+    <string name="home_popular_section_title">Popular on Jetnews</string>
+    <string name="home_post_min_read">%1$s - %2$d min read</string>
+    <string name="home_post_based_on_history">BASED ON YOUR HISTORY</string>
+
+    //Article Screen
```
Comment (chrisbanes) [inline JetNews/app/src/main/res/values/strings.xml:41]: Ditto

File: JetNews/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -28,4 +28,25 @@
...
+    <string name="article_published_in">Published in: %s</string>
+    <string name="article_functionality_not_available">Functionality not available \uD83D\uDE48</string>
+    <string name="article_share_post">Share post</string>
+    <string name="article_post_min_read">%1$s • %2$d min read</string>
+
+    //Interests Screen
```
Comment (chrisbanes) [inline JetNews/app/src/main/res/values/strings.xml:47]: Ditto

**Review by chrisbanes — COMMENTED** [review]
LGTM, just a few things

File: JetNews/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (pandubaraja) [reply JetNews/app/src/main/res/values/strings.xml]: Ahh thanks for pointing 
Done!

File: JetNews/app/src/main/res/values/strings.xml (hunk 3 above)
Comment (pandubaraja) [reply JetNews/app/src/main/res/values/strings.xml]: Done too

File: JetNews/app/src/main/res/values/strings.xml (hunk 2 above)
Comment (pandubaraja) [reply JetNews/app/src/main/res/values/strings.xml]: Done too

**Review by chrisbanes — APPROVED** [review]

**Pull request merged**